ctapipe/reco/impact.py

```diff
@@ -85,51 +88,70 @@ class ImPACTReconstructor(Component):
 
     """
 
+    minimiser = traits.CaselessStrEnum(
+        ["minuit", "l-BFGS"],
+        default_value="minuit",
+        help="name minimiser to use in the fit",
+    ).tag(config=True)
+
+    atmosphere_profile_name = traits.CaselessStrEnum(
```

Comment:
This should be replaced with the atmosphere profile provided by the EventSource. That will ensure it cannot be out of sync.